versions/3.0.md

```diff
@@ -849,7 +849,7 @@ A path parameter of a string value:
 {
   "name": "username",
   "in": "path",
-  "description": "username to fetch",
+  "description": "user name to fetch",
```

Comment:
I'm no native speaker as well, but when it comes to these cases, I use a google search for the word (if needed, add a "define" before the word) and see if it comes up with a result. `username` is commonly used today, and there's no reason to break it down.